pyvo/astropy_samp/hub_script.py

```diff
@@ -0,0 +1,144 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+
+import copy
+import time
+import sys
+import argparse
+
+from astropy import log
+
+from .hub import SAMPHubServer
+
+__all__ = ['hub_script']
+
+
+def hub_script(timeout=0):
+    """
+    This main function is executed by the ``samp_hub`` command line tool.
+    """
+
+    # TODO: Currently defines astropy version from which this is copied over.
+    # PyVO should define a new versioning scheme for this module.
+    parser = argparse.ArgumentParser(prog="samp_hub 4.2")
```

Comment:
Also let me know how you want to track its version. It was using `astropy.__version__` but I changed it to 4.2. If I use `pyvo.__version__`, it will be too confusing because `pyvo` still at 1.x.